src/class-wc-gateway-btcpay.php

```diff
@@ -621,7 +627,8 @@ public function process_payment($order_id)
                 throw new \Exception('The BTCPay payment plugin was called to process a payment but could not retrieve the order details for order_id ' . $order_id . '. Cannot continue!');
             }
 
-            $notification_url = $this->get_option('notification_url', WC()->api_request_url('WC_Gateway_BtcPay'));
+            //$notification_url = $this->get_option('notification_url', WC()->api_request_url('WC_Gateway_BtcPay'));
+            $notification_url = get_home_url() . '/wp-json/btcpay/ipn/status';
```

Comment:
Are you sure about this?

Reply:
I don't think there's a benefit in letting plugin users configure the IPN url. So using get_home_url() is  a safe option imo.